src/main/java/uk/gov/digital/ho/hocs/info/api/MemberService.java

```diff
@@ -42,28 +46,50 @@ Member getMember(UUID uuid) {
         }
     }
 
-    void updateWebMemberLists() {
+    public Integer updateWebMemberLists() {
         log.info("Started Updating Members Lists");
-        updateMember(listConsumerService.createFromWelshAssemblyAPI());
-        updateMember(listConsumerService.createFromScottishParliamentAPI());
-        updateMember(listConsumerService.createCommonsFromUKParliamentAPI());
-        updateMember(listConsumerService.createLordsFromUKParliamentAPI());
-        updateMember(listConsumerService.createFromIrishAssemblyAPI());
+        Long start = System.currentTimeMillis();
```

Comment:
Is this valid for when in an environment or was it just for local testing?

Reply:
Yes that can go, wanted to collect metrics during dev and would like to know how it behaves in prod but it is not essential.

Reply:
Could use the log above timestamp with the finishing one for a rough guess. Won't be fully accurate though...

Reply:
I think it should be removed; it served its purpose, the original refresh took approx 24 seconds and this version takes about 7 seconds.